python/tank/platform/application.py

```diff
@@ -162,6 +163,29 @@ def log_exception(self, msg):
         self.engine.log_exception(msg)
 
 
+    ##########################################################################################
+    # internal API
+
+    def log_metric(self, action):
+        """Logs an app metric.
+
+        :param action: Action string to log, e.g. 'Execute Action'
+
+        Logs a user activity metric as performed within an app. This is a
+        convenience method that auto-populates the module portion of
+        `tank.util.log_user_activity_metric()`.
+
+        Internal Use Only - We provide no guarantees that this method
+        will be backwards compatible.
+
+        """
+        # the action contains the engine and app name, e.g.
+        # module: tk-multi-loader2
+        # action: (tk-maya) tk-multi-loader2 - Load...
+        full_action = "(%s) %s - %s" % (self.engine.name, self.name, action)
```

Comment:
I took it out of each of these wrapper methods. As I test these changes from CR, I'll see how it feels in totango.